components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -1703,6 +1695,22 @@
         }
     }
 
+    function validateAttribute(attributeName) {
+        var attribute = document.getElementsByName(attributeName)[0];
+        var error = "";
+        var value = attribute.value;
+        if (value.length == 0) {
+            error = attribute.name + " ";
+            return error;
+        }
+        value = value.replace(/^\s+/, "") ;
+        if (value.length == 0) {
+            error = attribute.name + "(contains only spaces) ";
```

Comment:
add a space before (contains

Also why we need to enclose the error message within brackets?